### ISSUE TITLE
How to install your zip file on Linux Ubuntu ?

### ISSUE THREAD

**Issue by AlexisPERROTTEY** [Zenahr/lichess-desktop-app#8]
Hi Zenahr, your linux64.zip doesn't come with any README or INSTALL instruction.
How to install your zip file on Linux Ubuntu v24 ?

**Comment by thusharkn** [comment]
Yeah, have the same doubt. I tried running the lichess executable in the parent folder, but apturl couldn't find any link to install it from.

**Comment by moenie99** [comment]
for me, it works just fine to download the archive, unzip it and give permission for the file called lichess to be executable
